test/integration/fixtures/regression/issue-2286.js

```diff
@@ -0,0 +1,5 @@
+describe('suite', function () {
+  beforeEach(function () { this.skip() })
+  after(function () { console.log('after in suite') })
+  it('test', function () {})
+})
```

Comment:
How about some names that make this file's function obvious (both in viewing this file and if it were to end up run among the main tests in addition to the meta test calling it by `run`)? e.g.:
describe: 'testception for issue 2286 meta test'
beforeEach: 'skips tests'
after: 'should run'
it: 'skipped by beforeEach'